jsonschema-core.xml

```diff
@@ -1587,206 +1587,75 @@
+                                current base URI, in the same manner as for "$ref".
                             </t>
                             <t>
-                                In order to create a recursive reference, we must do three things:
-                                <list>
-                                    <t>
-                                        In our original schema, indicate that the schema author
-                                        intends for it to be extensible recursively.
-                                    </t>
-                                    <t>
-                                        In our extension schema, indicate that it is intended
-                                        to be a recursive extension.
-                                    </t>
-                                    <t>
-                                        Use a reference keyword that explicitly activates the
-                                        recursive behavior at the point of reference.
-                                    </t>
-                                </list>
-                                These three things together ensure that all schema authors
-                                are intentionally constructing a recursive extension, which in
-                                turn gives all uses of the regular "$ref" keyword confidence
-                                that it only behaves as it appears to, using lexical scoping.
+                                The schema identified by the resulting URI is examined for the
+                                presence of "$recursiveAnchor", and a new base URI is calculated
+                                as described for that keyword in the following section.
                             </t>
                             <t>
-                                The "$recursiveAnchor" keyword is how schema authors indicate
-                                that a schema can be extended recursively, and be a recursive
-                                schema.  This keyword MAY appear in the root schema of a
-                                schema document, and MUST NOT appear in any subschema.
+                                Finally, the value of "$recursiveRef" is resolved against the
+                                new base URI determined according to "$recursiveAnchor" producing
+                                the final resolved reference URI.
                             </t>
                             <t>
-                                The value of "$recursiveAnchor" MUST be of type boolean, and
-                                MUST be true.  The value false is reserved for possible future use.
+                                Note that in the absence of "$recursiveAnchor" (and in some cases
+                                when it is present", "$recursiveRef"'s behavior is identical to
```

Comment:
Assuming there should be a closing bracket here?

Reply:
Heh, you don't like infinite asides?

Reply:
Fixed.